src/graphics/render_window.rs

```diff
@@ -258,6 +474,19 @@ impl RenderWindow {
     ///
     /// If set, grabs the mouse cursor inside this window's client area so it may no longer be
     /// moved outside its bounds. Note that grabbing is only active while the window has focus.
+    ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// window.set_mouse_cursor_visible(true);
```

Comment:
You're calling `set_mouse_cursor_visible` here, not `set_mouse_cursor_grabbed`

Reply:
This is part of the small self-explanatory examples that aren't needed. Decided to get rid of this example entirely